Docker-compose-web-srv-https-redirect/SyncFldr/debian-apache2/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:22.10
+
+RUN apt-get update -y; \
+    apt-get install -y apt-utils; \
```

Comment:
use one apt-get install instead of 3 like you did in Docker-compose-web-srv-HAProxy/SyncFldr/apache2-adm/Dockerfile